Make the fact that _default is optional more obvious (#2560)
PX4/PX4-user_guide: en/dev_setup/building_px4.md

To build for NuttX- or Pixhawk- based boards, navigate into the **PX4-Autopilot** directory and then call `make` with the build target for your board.

For example, to build for [Pixhawk 4](../flight_controller/pixhawk4.md) hardware you could use the following command:

```sh
cd PX4-Autopilot
make px4_fmu-v5_default
```

A successful run will end with similar output to:

```sh
-- Build files have been written to: /home/youruser/src/PX4-Autopilot/build/px4_fmu-v4_default
[954/954] Creating /home/youruser/src/PX4-Autopilot/build/px4_fmu-v4_default/px4_fmu-v4_default.px4
```

> The first part of the build target `px4_fmu-v4` indicates the target flight controller hardware for the firmware

Author: ```suggestion
The first part of the build target `px4_fmu-v4` indicates the target flight controller hardware for the firmware.
```

The suffix, in this case `_default`, indicates a firmware _configuration_, such as supporting or omitting particular features.

:::note

> The `_default` suffix is optional.
> For example, `make px4_fmu-v5` and `px4_fmu-v5_default` are equivalent.

Author: ```suggestion
The `_default` suffix is optional.
For example, `make px4_fmu-v5` and `px4_fmu-v5_default` result in the same firmware.
```
